src/services/ui/src/pages/dashboard/Lists/renderCells/index.tsx

```diff
@@ -47,6 +48,22 @@ export const renderCellActions = (user: CognitoUserAttributes | null) =>
           <POP.PopoverContent>
             <div className="flex flex-col">
               {actions.map((action, idx) => {
+                if (data.authority === "WAIVER") {
+                  return (
+                    <TypedLink
+                      path="/action/waiver/:id/:type"
+                      key={`${idx}-${action}`}
+                      params={{ id: data.id, type: action }}
+                      className={cn(
+                        "text-blue-500",
+                        "relative flex select-none items-center rounded-sm px-2 py-2 text-sm outline-none transition-colors hover:bg-accent hover:text-accent-foreground data-[disabled]:pointer-events-none data-[disabled]:opacity-50"
+                      )}
+                    >
+                      {/* /action/waiver/someid/sometype */}
+                      {mapActionLabel(action)}
+                    </TypedLink>
+                  );
+                }
                 return (
                   <Link
                     className={cn(
```

Comment:
Playing devils advocate, here's an alternative which collapses the business logic into one render:
```js
<Link
  className={cn(
    "text-blue-500",
    "relative flex select-none items-center rounded-sm px-2 py-2 text-sm outline-none transition-colors hover:bg-accent hover:text-accent-foreground data-[disabled]:pointer-events-none data-[disabled]:opacity-50"
  )}
  path={(() => {
    if (data.authority === "WAIVER") {
      return "/action/waiver/:id/:type";
    }
    return "/action/:id/:type";
  })()}
  params={{ id: data.id, type: action }}
  {...(data.authority !== "WAIVER" && {
    origin: "actionsDashboard",
  })}
  key={`${idx}-${action}`}
  aria-label={action}
>
  {mapActionLabel(action)}
</Link>
```